content/docs/user-guide/editor/asset-browser.md

```diff
@@ -0,0 +1,156 @@
+     ![Asset Browser Edit.](/images/user-guide/assetbrowser/breadcrumbs-edit.png)
+
+## Opening Assets
+
+1. Using the **Asset Browser**, you can open and edit assets by double clicking the **Left Mouse Button** on an asset.
+
+- To open your asset in the editor, drag and drop your asset from the **Asset Browser** into the **Entity Outliner** or **Viewport**.
+- To open your asset with an associated application, use **Right Mouse Button** on an asset to bring up the **context menu**, choose **Open with associated application...**
+- To view your asset in your operating system's **File Explorer**, use **Right Mouse Button** to bring up the **context menu** and choose **Open in Explorer**.
+
+## Search and Filter Assets
+
+For projects that have many assets, you can search and filter for the assets that you want. Enter text in the search filter box to find specific assets.
+
+**Search for assets and use type filters**
+
+1. In the **Asset Browser**, enter the name of the asset.
+
+1. To create a filter by **Asset Type**, click the filter icon.
+
+1. Enter the asset type in the search field that appears or scroll and select the asset type that you want. Any asset that has that specified asset type appears in the results.
+
+     The assets that appear are either **Material** assets or **Physics Material** assets.
+
+     ![Asset Browser Asset Type Filter.](/images/user-guide/assetbrowser/asset-type-filter.png)
+
+1. To clear search results, click **Clear**.
+
+**Advanced asset filter options**
+
+1. In the **Asset Browser**, open the **Asset Browser Menu**
+
+     ![Asset Browser Advanced Filter Options.](/images/user-guide/assetbrowser/advanced-filter-options.png)
+
+- Uncheck **Hide Engine Folders** to display assets that are not inside of your project folder.
+     
+     In the current release, this must be unchecked to view any external directories that have been added, including **Asset Gems**.
+
+- Uncheck **Hide Unusable Product Assets** to display assets that are not usable by the editor.
+
+     You can add or remove **Unusable Product Assets** using the  [Settings Registry System](/docs/user-guide/settings) by creating a JSON with a `.setreg` extension.
+
+     In your new JSON file, specify the desired **Product Asset** along with its **Uuid**, and use the following JSON path: `/O3DE/AssetBrowser/AssetTypeUuidExcludes`. **Example**
+
+     ```
+     {
+    "O3DE": {
+        "AssetBrowser": {
+            "AssetTypeUuidExcludes": {
+                "AzslOutcomeAsset":  "{6977AEB1-17AD-4992-957B-23BB2E85B18B}",
+                "ModelLodAsset": "{65B5A801-B9B9-4160-9CB4-D40DAA50B15C}",
+                "ImageMipChainAsset": "{CB403C8A-6982-4C9F-8090-78C9C36FBEDB}", 
+                "BufferAsset": "{F6C5EA8A-1DB3-456E-B970-B6E2AB262AED}", 
+                "ShaderVariantAsset": "{51BED815-36D8-410E-90F0-1FA9FF765FBA}"
+                                     }
+                        }
+            }
+     }
+     ```
```

Comment:
Restructured the numbered steps and the bullet points
````suggestion
### Advanced asset filter options

1. In the **Asset Browser**, open the **Asset Browser Menu**

    ![Asset Browser Advanced Filter Options.](/images/user-guide/assetbrowser/advanced-filter-options.png)

2. Use the following options as needed:

   - Disable **Hide Engine Folders** to display assets that are not inside of your project folder.
     
        This must be disabled to view any external directories that you may have added, including **Asset Gems**.

   - Disable **Hide Unusable Product Assets** to display assets that the Editor cannot use.

        You can add or remove **Unusable Product Assets** using the  [Settings Registry System](/docs/user-guide/settings) by creating a JSON with a `.setreg` extension.

        In your new JSON file, specify the desired **Product Asset** along with its **UUID**, and use the following JSON path: `/O3DE/AssetBrowser/AssetTypeUuidExcludes`. 
        
        **Example**

        ```json
        {
        "O3DE": {
            "AssetBrowser": {
                "AssetTypeUuidExcludes": {
                    "AzslOutcomeAsset":  "{6977AEB1-17AD-4992-957B-23BB2E85B18B}",
                    "ModelLodAsset": "{65B5A801-B9B9-4160-9CB4-D40DAA50B15C}",
                    "ImageMipChainAsset": "{CB403C8A-6982-4C9F-8090-78C9C36FBEDB}", 
                    "BufferAsset": "{F6C5EA8A-1DB3-456E-B970-B6E2AB262AED}", 
                    "ShaderVariantAsset": "{51BED815-36D8-410E-90F0-1FA9FF765FBA}"
                                        }
                            }
                }
        }
        ```
````